storage/block/block.go

```diff
@@ -287,15 +288,15 @@ func (b *dbBlock) resetMergeTargetWithLock() {
 }
 
 type databaseSeriesBlocks struct {
-	elems map[time.Time]DatabaseBlock
+	elems map[xtime.UnixNano]DatabaseBlock
 	min   time.Time
```

Comment:
could make min/max `UnixNano` too

Reply:
I tried changing them and they're actually exposed in public interfaces. Might be best to just keep them as time.Time for now. I did a bunch of benchmarking and even with the conversions back and forth, this code should be faster https://gist.github.com/richardartoul/2ce3c5258df0fb6f47a030950eafe93e